### PR TITLE
KARAF-3982: Add support for editing property files to the assembly mojo.

### PR DESCRIPTION
**Pull request by bimargulies**
This appears to be all done.

### CODE REVIEW

**Comment by maggu2810** [comment]
Hello, what do you think about the append / prepend hint?

**Comment by bimargulies** [comment]
Done

File: tooling/utils/pom.xml [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+        See the License for the specific language governing permissions and
+        limitations under the License.
+    -->
+
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <groupId>org.apache.karaf.tooling</groupId>
+        <artifactId>tooling</artifactId>
+        <version>4.0.2-SNAPSHOT</version>
```
Comment (maggu2810) [inline tooling/utils/pom.xml]: This should be changed to match the project version of the current master.

**Comment by maggu2810** [comment]
Hi @bimargulies,
what about variable substitution? Using e.g. ${project.version} e.g. in an added value for a property seems a valid and perhaps common use case. What do you think about?

**Comment by maggu2810** [comment]
e.g. a attribute for the value tag "substitue" to switch the substitution (InterpolationHelper.substVars(...)) on / off.

**Pull request merged**